modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/model/listener/test/APIEndpointRelevantObjectEntryModelListenerTest.java

```diff
@@ -130,7 +130,28 @@ public void test() throws Exception {
 				"path",
 				StringBundler.concat(
 					RandomTestUtil.randomString(), StringPool.FORWARD_SLASH,
-					RandomTestUtil.randomString(), StringPool.COMMA)
+					StringPool.COMMA)
+			).put(
+				"scope", "company"
+			).toString(),
+			"headless-builder/endpoints", Http.Method.POST);
+
+		Assert.assertEquals("BAD_REQUEST", jsonObject.get("status"));
+		Assert.assertEquals(
+			"Path must start with the \"/\" character",
+			jsonObject.get("title"));
+
+		jsonObject = HTTPTestUtil.invokeToJSONObject(
```

Comment:
This is a Brian's request :smile:  but he wants error messages sorted. I mean if you are testing a test case, its error message should be alphabetically ordered. Write this test case down